### PR TITLE
[plugin.video.openingsmoe] 2.3.1

### PR DESCRIPTION
**Pull request by Rucadi**
### Description
Openings.moe Unofficial Kodi Plugin.
You can reproduce randomized http://openings.moe/ videos.
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.openingsmoe/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+name="OpeningsMoe"
+provider-name="Rucadi">
+<requires>
+  <import addon="xbmc.python" version="2.25.0"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en">OpeningsMoe viewer for KODI</summary>
```
Comment (enen92) [inline plugin.video.openingsmoe/addon.xml]: en_GB

File: plugin.video.openingsmoe/main.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+
+def playVideo(jslist):
+    pl = xbmc.PlayList(xbmc.PLAYLIST_VIDEO)
+    pl.clear()
+    for u in jslist:
+        pl.add("http://openings.moe/video/"+u['file'], xbmcgui.ListItem(u['source']+" "+u['title']))
+    pl.shuffle()
+    xbmc.Player().play(pl, windowed=False)
```
Comment (enen92) [inline plugin.video.openingsmoe/main.py]: plugins should not use `xbmc.Player()` (plugins only set the resolved url on listitems). Custom player instances should be only used in scripts

File: plugin.video.openingsmoe/main.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import sys
+from urllib import urlencode
+from urlparse import parse_qsl
+import xbmcgui
```
Comment (enen92) [inline plugin.video.openingsmoe/main.py]: please remove unused imports. You don't use `xbmcgui` and `xbmcplugin`. The fact you don't use `xbmcplugin` is a pretty good indicator your addon is not a plugin but a script.

**Comment by enen92** [comment]
ping @Rucadi. I'll close this in the next days if no ack is given. The PR is sitting here for too long.

**Comment by Rucadi** [comment]
Sorry, I totally forgot about this. 

I just changed the things you said and will proceed to PR on the repo-scripts.

**Comment by enen92** [comment]
Ok thanks much

**Pull request closed without merging**